### DISCUSSION TITLE
Finished the JavaScript assignment

### DISCUSSION THREAD

**Discussion by gman193** [ccci-code/summerproject#11] [General]
I finished the JavaScript assignment a while ago. What would you like me to do next?

**Comment by ccci-code** [comment]
Great!  I suggest learning how to use random numbers in Javascript (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)

Assignment:

Write a Javascript function that returns a random main, side, and vegetable using the data in meals.json which is loaded into the variable "meals"

Use the jsontester.html page where I've added a button to run the function you will write called "randomMeal()"

Return the results in an object, ie:  {"main":"hamburgers","side":"french fries","vegetable":"corn"}
